roles/maintenance_10_linux/tasks/main.yml

```
@@ -72,7 +72,7 @@
   ansible.builtin.find:
     paths:
       - /var/log
-    size: 16m
+    size: "{{ linux_allowed_large_logfiles_maxsize_mb }}"
```

Is the trailing `m` optional? if not, please add it:

```suggestion
    size: "{{ linux_allowed_large_logfiles_maxsize_mb }}m"
```